src/tilemaker/entrypoint.sh

```diff
@@ -7,6 +7,8 @@ source /opt/pfb/tilemaker/scripts/utils.sh
 
 export TL_MIN_ZOOM="${TL_MIN_ZOOM:-8}"
 export TL_MAX_ZOOM="${TL_MAX_ZOOM:-18}"
+export TL_NUM_PARTS="${TL_NUM_PARTS:-4}"
```

Comment:
So it runs one `tl` command per partition per zoomlevel (for 13 and above. below that I judged there aren't enough tiles for it to be worth partitioning), so with max-zoom 17 that's 20 total jobs.  The higher-zoomlevel ones will still be significantly bigger than the lower-zoomlevel ones, though.  We could make the partition count scale with zoomlevel, either linearly to mitigate the difference somewhat or exponentially to make all the jobs about the same size.  Might make sense to partition in both directions in that case, too.  I just did one way for simplicity.